### ISSUE TITLE
Add tips to help new users

### ISSUE THREAD

**Issue by rdmpage** [gbif/portal16#311]
I've been stuck by how some sites use tips effectively to help new users. For example, sloppy.io uses a service called [AppCues](https://www.appcues.com) to guide the user around:

![screenshot 2017-03-12 10 55 41](https://cloud.githubusercontent.com/assets/83306/23831134/b1912324-0712-11e7-89d0-284f6c28957a.png)

AppCues is not free, but there are some open source alternatives, such as [shepherd](https://github.com/HubSpot/shepherd):
![screenshot 2017-03-12 10 56 01](https://cloud.githubusercontent.com/assets/83306/23831146/e799abf8-0712-11e7-98fe-ac0fd1c451fb.png)

For new users there's a lot of jargon and potential complexity to get one's head around when looking at the GBIF web site, I suspect we could reduce that by some nice tips and "stories".

**Comment by kbraak** [comment]
I agree some tips or an interactive guide aimed at new users would help boost proper usage of GBIF.org. Some push back on this idea, saying the interface is (or aims to be) easy enough to use that tips aren't required.

The IPT effectively uses tips throughout its interface to make it easier to use. A screenshot below from the IPT below demonstrates how it looks. 

Note a glossary has also be proposed to store explanations for all that jargon - see https://github.com/gbif/portal16/issues/240 

![screen shot 2017-03-13 at 13 04 09](https://cloud.githubusercontent.com/assets/5981100/23853752/582abeee-07ee-11e7-9bf1-b003dfca975c.png)

**Comment by rdmpage** [comment]
@kbraak 

> Some push back on this idea, saying the interface is (or aims to be) easy enough to use that tips aren't required.

I suspect that those of us who use GBIF a lot know out way around, those who are new or irregular users may no find it quite so intuitive. Be great to observe, say, some undergraduate students using it and see how they get on.

**Comment by MortenHofft** [comment]
> Some push back on this idea, saying the interface is (or aims to be) easy enough to use that tips aren't required.

@kbraak I think we might have misunderstood each other then. I am by no means adverse to help boxes, tutorials, guides etc in any form. I would love to add help to occurrence search for example. I also suggested that a glossary of terms should be written a year ago when I just started using the site. But it is true, that I would think it a shame to rely on it as the go to solution. Many of the usability issues we have on the current site I _hope_ have been solved on the new site without popups galore. 

That said I have no doubt introduced new issues as well. And other stuff where I don't know how to simplify it, so helptexts is probably what we will end up with. 

Sorry if I have seemed dismissive of such suggestions - that haven't been my intention.

**Comment by kbraak** [comment]
Thanks @MortenHofft, 

It may interest you to see the simple approach used in the [new Barcode of Life Portal](http://v4.boldsystems.org/index.php/Public_BINSearch?searchtype=records). Clicking "Show Help" displays the help text within the search page, meaning the interface isn't cluttered with a lot of popups and the user doesn't have to be redirected to another page. Screenshots below are included for your convenience. 

### Screenshots
**Before:**
![screen shot 2017-05-03 at 16 07 40](https://cloud.githubusercontent.com/assets/5981100/25665090/ded29240-301c-11e7-95d8-f4a617581845.png)
**After:**
![screen shot 2017-05-03 at 16 08 14](https://cloud.githubusercontent.com/assets/5981100/25665110/edcb5bc4-301c-11e7-96f8-f80e024bb309.png)
